<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SwitchWidget</class>
 <widget class="QWidget" name="SwitchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>186</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>296</width>
    <height>186</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TouchmatModeWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>11</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="textWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>240</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
</string>
     </property>
     <widget class="QLabel" name="lampOff">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>112</y>
        <width>240</width>
        <height>56</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
        font-family: SegoeUI;
    font-size: 20px;
    font-weight: normal;
    font-style: normal;
        color: #e7e7e7;
    padding-right: 30px;
}</string>
      </property>
      <property name="text">
       <string>Lamp Off</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lampOn">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>56</y>
        <width>240</width>
        <height>56</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
        font-family: SegoeUI;
    font-size: 20px;
    font-weight: normal;
    font-style: normal;
        color: #e7e7e7;
    padding-right: 30px;
}</string>
      </property>
      <property name="text">
       <string>Lamp On</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="desktop">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>240</width>
        <height>56</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
        font-family: SegoeUI;
    font-size: 20px;
    font-weight: normal;
    font-style: normal;
        color: #e7e7e7;
    padding-right: 30px;
}</string>
      </property>
      <property name="text">
       <string>Desktop</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="buttonWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>168</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #4a4e52;
border-radius: 28px;
</string>
     </property>
     <widget class="QPushButton" name="lightOffButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>112</y>
        <width>56</width>
        <height>56</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>56</width>
        <height>56</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>56</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    image: url(:/resources/icon-lightoff-norm.png);
	color: #e7e7e7;
    background-color: transparent;
    border: none;
	image-position: center center;
	background-repeat: none;
}

QPushButton:hover {
    image: url(:/resources/icon-lightoff-hover.png);
	color: #ffffff;
 }

QPushButton:disabled {
	image: url(:/resources/icon-lightoff-disabled.png);
	color: #64686C;
}

QPushButton:pressed {
	image: url(:/resources/icon-lightoff-press.png);
	color: #0096D6;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="lightOnButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>56</y>
        <width>56</width>
        <height>56</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>56</width>
        <height>56</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    image: url(:/resources/icon-lighton-norm.png);
	color: #e7e7e7;
    background-color: transparent;
    border: none;
	image-position: center center;
	background-repeat: none;
}

QPushButton:hover {
    image: url(:/resources/icon-lighton-hover.png);
	color: #ffffff;
 }

QPushButton:disabled {
	image: url(:/resources/icon-lighton-disabled.png);
	color: #64686C;
}

QPushButton:pressed {
	image: url(:/resources/icon-lighton-press.png);
	color: #0096D6;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="desktopButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>56</width>
        <height>56</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>56</width>
        <height>56</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    image: url(:/resources/icon-monitor-norm.png);
	color: #e7e7e7;
    background-color: transparent;
    border: none;
	image-position: center center;
	background-repeat: none;
}

QPushButton:hover {
    image: url(:/resources/icon-monitor-hover.png);
	color: #ffffff;
 }

QPushButton:disabled {
	image: url(:/resources/icon-monitor-disabled.png);
	color: #64686C;
}

QPushButton:pressed {
	image: url(:/resources/icon-monitor-press.png);
	color: #0096D6;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="highLight" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>112</y>
        <width>56</width>
        <height>56</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #0096d6;
border-radius: 28px;</string>
      </property>
     </widget>
     <zorder>highLight</zorder>
     <zorder>lightOffButton</zorder>
     <zorder>lightOnButton</zorder>
     <zorder>desktopButton</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
